RFCs/0031-Multi-Test-Run-Finalization.md

```diff
@@ -0,0 +1,137 @@
+# 0031 Multi Test Run Finalization
+
+# Summary
+Introduce APIs to perform data attachments reprocessing (e.g. combining, merging) after two or more independent test executions.
```

Comment:
Perhaps we could highlight the scenario a little better. 
Suggestion: "This document details a data collector extensibility point to reporcess (combine/merge) attachments obtained across test executions in both design mode and commandline scenarios"

I'm assuming we refer to the non-commandline mode as "design mode". Also please consider using `data collector attachments` instead.